apps/web/app/components/features/features.css

```
@@ -0,0 +1,310 @@
+    display: flex;
+    gap: 12px;
+    width: max-content;
+    animation: scrollLeft 40s linear infinite;
+    padding: 4px 0;
+}
+
+.vuln-strip-track:hover {
+    animation-play-state: paused;
+}
+
+@keyframes scrollLeft {
```

Stylelint (17.3.0)

[error] Expected keyframe name "scrollLeft" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)